index.html

```diff
@@ -3515,8 +3515,23 @@ <h2>
           <li>Return <var>promise</var> and perform the remaining steps <a>in
           parallel</a>.
           </li>
-          <li>Close down any remaining user interface. The <a>user agent</a>
-          MAY use the value <var>result</var> to influence the user experience.
+          <li>If <var>document</var> stops being <a data-cite=
+          "!HTML#fully-active">fully active</a> while the user interface is
+          being shown, or no longer is by the time this step is reached, then:
+            <ol>
+              <li>Close down the user interface.
+              </li>
+              <li>Set the <a>user agent</a>'s <a>payment request is showing</a>
+              boolean to false.
+              </li>
+              <li>Reject <var>promise</var> with an "<a>AbortError</a>"
+              <a>DOMException</a>.
+              </li>
+            </ol>
+          </li>
+          <li>Otherwise, close down any remaining user interface. The <a>user
```

Comment:
Maybe all three of these steps should be under "otherwise"? Doesn't actually have normative impact, but seems cleaner.